### PR TITLE
Companion PR for #4954

### PR DESCRIPTION
**Pull request by seunlanlege**

### CODE REVIEW

**Comment by gavofyork** [comment]
Companion PRs should not alter the branch.

**Comment by seunlanlege** [comment]
> Companion PRs should not _alter_ the branch.

Hmm, I'm not really sure what you mean. shouldn't merge with master? shouldn't add new commits?

File: availability-store/Cargo.toml [hunk 1]
```diff
@@ -15,14 +15,14 @@ futures = "0.3.4"
...
 codec = { package = "parity-scale-codec", version = "1.3.0", features = ["derive"] }
-sc-network = { git = "https://github.com/paritytech/substrate", branch = "master" }
-consensus_common = { package = "sp-consensus", git = "https://github.com/paritytech/substrate", branch = "master" }
-client = { package = "sc-client-api", git = "https://github.com/paritytech/substrate", branch = "master" }
-sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master" }
-sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "master" }
-sp-api = { git = "https://github.com/paritytech/substrate", branch = "master" }
-keystore = { package = "sc-keystore", git = "https://github.com/paritytech/substrate", branch = "master" }
-sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sc-network = { git = "https://github.com/paritytech/substrate", branch = "seun-sc-cli-subkey" }
```
Comment (rakanalh) [inline availability-store/Cargo.toml:18]: branch should always be "master"

Comment (seunlanlege) [reply availability-store/Cargo.toml]: ah thanks for this :pray:

**Comment by seunlanlege** [comment]
setting branch as master breaks CI:

```
$ time cargo test --all --release --verbose --locked --features runtime-benchmarks
    Updating git repository `https://github.com/paritytech/substrate`
error: no matching package named `frame-utils` found
location searched: https://github.com/paritytech/substrate
required by package `kusama-runtime v0.8.1 (/builds/parity/polkadot/runtime/kusama)`
```

**Comment by seunlanlege** [comment]
going to go ahead and revert it, so i can confirm that CI passes.

**Comment by kianenigma** [comment]
status @seunlanlege ?

**Comment by seunlanlege** [comment]
looks like no change was needed after all

**Pull request closed without merging**